### ISSUE TITLE
Metabase Can't Display Two Foreign Key Lookups

### ISSUE THREAD

**Issue by exabrial** [metabase/metabase#11626]
**Describe the bug**
We have a `code_values` as a repository of commonly used strings. We have another table that has two 
pointers to that table. In the "Data Model" settings, if we go to "Display Values" and set the first column to "Use Foreign Key", metabase will display the String of the Foreign key. If we repeat the process for the second column, it works for the Second column, but the first column reverts to displaying the ID number, rather than the foreign key. 

Basically, Metabase can only handle on Foreign Key lookup at a time per table. A table with two pointers seems to cause trouble.

Metabase v34 (latest at the time), Ubuntu Linux 18.04, AdoptOpenJDK 1.8, Brave Browser.

**Comment by flamber** [comment]
Hi @exabrial 
Sounds like a duplicate of #9236. If you agree, then please close this and upvote the other issue :+1:

**Comment by mazameli** [comment]
Yeah, seems like a dupe. Closing as such.
